sdk/src/linux/usb_device_linux.cpp

```diff
@@ -657,10 +657,6 @@ aditof::Status UsbDevice::applyCalibrationToFrame(uint16_t *frame,
     float gain = m_implData->calibration_cache[mode].gain;
     float offset = m_implData->calibration_cache[mode].offset;
 
-    if (equal(gain, 1.0f) && equal(offset, 0.0f)) {
```

Comment:
What about the other implementations (ethernet, local, usb-windows)?

Reply:
My bad. Forgot that this is implemented in each device. Thanks!